src/gui/gt_icons.h

```diff
@@ -50,7 +119,7 @@ GT_GUI_EXPORT QIcon getIcon(const QString& iconPath);
  *  @param colorData Color data for the icon modes.
  * @return Icon
  */
-GT_GUI_EXPORT QIcon colorize(const QIcon& icon, SvgColorData colorData);
+GT_GUI_EXPORT QIcon colorize(const gt::gui::Icon& icon, SvgColorData colorData);
```

Comment:
I'd use `Icon` here as well, as you did everywhere else in this file
```suggestion
GT_GUI_EXPORT QIcon colorize(const Icon& icon, SvgColorData colorData);
```